packages/components/avatar-editor/sub-components/avatar-editor-body.js

```diff
@@ -331,10 +331,13 @@ class AvatarEditorBody extends React.Component {
 
     return (
       <Text as="span" textAlign={!desktopMode ? labelAlign : "left"}>
-        <Link type="action" fontWeight={600} isHovered {...onClickProp}>
-          {selectNewPhotoLabel}
-        </Link>{" "}
-        {desktopMode && orDropFileHereLabel}
+        <Box displayProp="flex" alignItems="center">
+          <Link type="action" fontWeight={600} isHovered {...onClickProp}>
+            {selectNewPhotoLabel}
+          </Link>
+          &nbsp;
```

Comment:
Check if a space is required for this link

Reply:
A space is required here to separate two text blocks in one line. (selectNewPhotoLabel and orDropFileHereLabel) Old type of space {" "} will not have effect here because it will be removed automatically inside flexbox container.